### ISSUE TITLE
Splash screen logo is too big, downscaling artefacts are clearly visible.

### ISSUE THREAD

**Issue by deviap-dev** [deviap/deviap-main#73]
Splash screen logo appears pixelated due to downscaling.

Solution is to save a smaller anti aliased version of the logo. preferably at correct dimensions.

**Comment by Sanjay-B** [comment]
Has this already been addressed?

**Comment by deviap-dev** [comment]
No. New image needs to be created so that the engine doesn’t have to rescale the graphics at runtime
